lib/fuse_lowlevel.c

```diff
@@ -2157,6 +2189,10 @@ void do_init(fuse_req_t req, fuse_ino_t nodeid, const void *inarg)
 		outargflags |= FUSE_SETXATTR_EXT;
 	if (se->conn.want & FUSE_CAP_DIRECT_IO_ALLOW_MMAP)
 		outargflags |= FUSE_DIRECT_IO_ALLOW_MMAP;
+	if (se->conn.want & FUSE_CAP_PASSTHROUGH) {
+		outargflags |= FUSE_PASSTHROUGH;
+		outarg.max_stack_depth = se->conn.max_backing_stack_depth + 1;
```

Comment:
Could you please add a comment here why it is +1?

Reply:
done